From 66f70e5da67f2a2ae60cc6b4d9babda7999743fe Mon Sep 17 00:00:00 2001
From: emC5RgViIK <gitlab.com@online.posch.name>
Date: Mon, 11 May 2020 16:23:04 +0000
Subject: [PATCH] (Cherry-picked from 47297402d38cafefd54310ee7b536b60c8524ac0)

---
 common/lib_tree_model.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/lib_tree_model.cpp b/common/lib_tree_model.cpp
index 71fe74c7d..ed30b11de 100644
--- a/common/lib_tree_model.cpp
+++ b/common/lib_tree_model.cpp
@@ -21,6 +21,7 @@
 
 #include <lib_tree_model.h>
 
+#include <algorithm>
 #include <eda_pattern_match.h>
 #include <lib_tree_item.h>
 #include <make_unique.h>
-- 
2.26.2

